daemon/cluster.go

```diff
@@ -6,6 +6,18 @@ import (
 
 // Cluster is the interface for github.com/docker/docker/daemon/cluster.(*Cluster).
 type Cluster interface {
+	ClusterStatus
+	NetworkManager
+}
+
+// ClusterStatus interface provides information about the Swarm status of the Cluster
+type ClusterStatus interface {
+	IsAgent() bool
+	IsManager() bool
+}
+
+// ClusterNetworkManager provides methods to manage networks
```

Comment:
oh! I was having an internal bike-shed over the name, decided that more "generic" was better. Let me update, thanks!